pysatMissions/instruments/pysat_ephem.py

```diff
@@ -119,21 +123,8 @@ def load(fnames, tag=None, sat_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
     if TLE2 is not None:
         line2 = TLE2
 
-    # grab date from filename
-    parts = os.path.split(fnames[0])[-1].split('-')
-    yr = int(parts[0])
-    month = int(parts[1])
-    day = int(parts[2][0:2])
-    date = pysat.datetime(yr, month, day)
-
-    # create timing at 1 Hz (for 1 day)
-    times = pds.date_range(start=date, end=date+pds.DateOffset(seconds=86399),
-                           freq='1S')
-    # reduce requirements if on testing server
-    # TODO Remove this when testing resources are higher
-    on_travis = os.environ.get('ONTRAVIS') == 'True'
-    if on_travis:
```

Comment:
Is the reduced number of samples on Travis no longer needed?

Reply:
The number of samples are now controlled in the unit tests.